1-mandatory/2-people-i-know.js

```diff
@@ -377,14 +377,36 @@ In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
```

Comment:
These all look good 👍 
I would recommend trying to write these again using array methods - trying to split each problem into multiple array methods.
This will be good practice, and will help you later in the course when you will see more array methods 😄